src/lib/component/partial/Label/Label.js

```diff
@@ -31,33 +49,89 @@ const styles = theme => ({
   },
 });
 
+const imageExtensions = [
+  ".PNG",
+  ".JPG",
+  ".JPEG",
+  ".GIF",
+  ".WEBP",
+  ".TIFF",
+  ".PSD",
+  ".RAW",
+  ".BMP",
+  ".HEIF",
+  ".INDD",
+  ".SVG.",
+];
+
 class Label extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     name: PropTypes.string.isRequired,
     value: PropTypes.string.isRequired,
   };
 
-  parseLink = value => {
+  parseLink = (value, name) => {
     try {
       new URL(value);
+      // see if it's got an image extension
+      let results = imageExtensions.filter(ext =>
+        value.toUpperCase().includes(ext),
+      );
+      let imageMatch = results.length > 0;
```

Comment:
To avoid having to check the `length` of the results. You could instead use the [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method instead of `filter`.